static/third/bootstrap-typeahead/typeahead.js

```diff
@@ -352,6 +356,12 @@
       return isSupported
     }
 
+  , resizeHandler: function() {
+      if(this.$container.css('display') == 'block') {
+        this.lookup();
+      }
```

Comment:
In particular, I'd like a comment here that explains to someone who doesn't know this code well why we're checking for `display: block` and why calling `lookup` is the right way to rerender.

(Another thing I worry about is whether this resize handler might pop open typeahead that was previously closed).

Reply:
Per my understanding, if `display:block` is present, then that means the typeahead is visible. Calling lookup() without any new input would just call the search codepath again, and end up in a rerender. It should be a comment.

Reply:
`lookup` does a lot of potentially expensive processing (computing the type of items to choose from, selecting those among them that match the query, sorting them, etc and constructing the menu element) apart from calling `show` which does exactly what we need - calculating and setting its position and finally showing it. So the call to `lookup` here should be replaced with `show`. Have tested this locally. Should be much more performant too!